plugins/meson.build

```diff
@@ -5,10 +5,15 @@ subdir('raspberrypi')
 subdir('steelseries')
 subdir('test')
 subdir('udev')
-subdir('unifying')
 subdir('upower')
 subdir('usb')
 
+if appstream_glib.version().version_compare('>=0.6.13')
+subdir('unifying')
+else
+message('appstream-glib 0.6.13+ not found, disabling unifying plugin')
```

Comment:
It does actually _compile_ with 0.6.9.

This goes back to https://github.com/hughsie/fwupd/pull/119.  

There wasn't much detail there, but it was actually referring back to `src/fu-main.c` using as_require_version_compare.  This was adjusted in appstream-glib 0.6.12: https://github.com/hughsie/appstream-glib/commit/a7ca77c563fd6898928ed4b083d8b36b477f45cc 
So #119 should have actually been 0.6.12 in the first place not 0.6.13.

So is that detrimental enough in all situations that you should only use unifying with 0.6.12+?